server_status/views_test.py

```diff
@@ -61,6 +63,32 @@ def test_view(self):
 
         self.assertTrue(resp["status_all"] == views.UP)
 
+    @data(
+        (None, views.DOWN, SERVICE_UNAVAILABLE),
+        (
+            "-----BEGIN CERTIFICATE-----MITX-----END CERTIFICATE-----",
+            views.DOWN,
+            SERVICE_UNAVAILABLE
+        )
+    )
+    @unpack
+    @override_settings(HEALTH_CHECK=['CERTIFICATE'])
+    def test_certificate_status_fail(self, certificate_loc, certificate_status, expected_status):
+        """
+        test app certificate expiry status.
+        """
+        with override_settings(MIT_WS_CERTIFICATE=certificate_loc):
+            resp = self.get(expected_status=expected_status)
+            assert resp['certificate']["status"] == certificate_status
+
+    def test_status_without_certificate(self):
+        """
+        test app when certificate path in not available in settings.
+        """
+        del settings.MIT_WS_CERTIFICATE
+        resp = self.get(expected_status=SERVICE_UNAVAILABLE)
+        assert 'certificate' not in resp
+
```

Comment:
Can you also add a `test_certificate_status_pass` test, with `@freeze_time("2019-01-18")` to make sure test passes assuming certificate has expiration date > 30 days after that.